core/src/com/group66/game/cannon/Cannon.java

```diff
@@ -30,34 +57,74 @@ else if (angle <= 0)
 			angle = 1;
 	}
 	
+	/**
+	 * Sets the angle of the cannon.
+	 *
+	 * @param angle the new angle
+	 */
 	public void setAngle(float angle) {
 		this.angle = angle;
```

Comment:
I added a commit that makes it so the maximum and minimum angle of the cannon are provided in the constructor, and the angles are now checked